lib/outputmanager.js

```diff
@@ -442,295 +448,244 @@ OutputPlugin.prototype.clearBuildFlag = function(pathToBuildFlag, next) {
 
 OutputPlugin.prototype.applyTheme = function(tenantId, courseId, jsonObject, destinationFolder, next) {
   var self = this;
+  var themeVariables = jsonObject.course[0].themeVariables;
+  var themeName = jsonObject['config'][0]._theme;
```

Comment:
Suggest mirroring the line above:
```suggestion
  var themeName = jsonObject.config[0]._theme;
```